core/jobs/batch_jobs/remove_profile_picture_data_url_field_jobs_test.py

```diff
@@ -66,9 +66,11 @@ def test_model_without_profile_picture_field_works(self) -> None:
         )
         self.assertNotIn('profile_picture_data_url', user_1.to_dict())
         self.assertNotIn(
-            'profile_picture_data_url', user_1._values)  # pylint: disable=protected-access
+            'profile_picture_data_url', user_1._values # fmt: skip # pylint: disable=protected-access, line-too-long
```

Comment:
Ditto -- explanation for fmt:skip, why do you need line-too-long.

Many places in this file.